graphql2/schema.graphql

```diff
@@ -289,10 +289,17 @@ type Mutation {
   updateUserOverride(input: UpdateUserOverrideInput!): Boolean!
   updateHeartbeatMonitor(input: UpdateHeartbeatMonitorInput!): Boolean!
 
+  updateAlertStatusByService(input: UpdateAlertStatusByServiceInput!): Boolean!
```

Comment:
I'm thinking we could call this `updateAlertsByService` to match `updateAlerts` which also only changes status.

Reply:
That sounds good. I couldn’t think of a good name so I went with the old API nomenclature 🙂